src/main/js/page_objects/blueocean/bluePipelines.js

```diff
@@ -27,5 +27,10 @@ module.exports.commands = [{
     },
     assertJob: function(jobName) {
         this.waitForElementVisible('.pipelines-table tr[data-name="' + jobName + '"]');
+    },
+    countJobToBeEqual: function(browser, jobName, count) {
```

Comment:
Do you need to pass `browser`? I think `this` will be the browser, so no need to pass it.